portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -97,7 +100,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         logger.warning("Could not convert Entity's dislikes to long"); 
       }
       
-      Comment comment = new Comment(id, user, text, timestamp, likes, dislikes);
+      Comment comment =
```

Comment:
Please add a todo for applying Builder Pattern to `Comment`

Reply:
Got it.